cypress/integration/spec.ts

```diff
@@ -0,0 +1,20 @@
+
+it('finds 1 result when auto-searching for "screwdriver"', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/senacor/simple-consumer/pullRequest?prid=7519855)
```suggestion
it("finds 1 result when auto-searching for \"screwdriver\"", () => {
```